src/components/App/index.js

```diff
@@ -1,19 +1,24 @@
 import React, { Component } from 'react';
 import { Switch, Route } from 'react-router-dom';
+import NotFound from '../NotFound/index';
+import Nav from '../Nav/index';
+import Assets from '../../container/Assets/index'
```

Comment:
We don’t need to put /index it is just assumed. So this can be ../../container/Assets, ../NotFound, ../Nav